00_Base/src/config/types.ts

```diff
@@ -79,6 +79,13 @@ export const systemConfigInputSchema = z.object({
         port: z.number().int().positive().default(8081).optional(),
       })
       .optional(),
+    tenant: z
+      .object({
+        endpointPrefix: z.string().default(EventGroup.Tenant).optional(),
+        host: z.string().default('localhost').optional(),
+        port: z.number().int().positive().default(8081).optional(),
```

Comment:
This might be a stupid questions, but wouldn't we want the default to be 8080? Why is the default 8081?

Reply:
if i'm understanding correctly, this is the config for the websocket which is why it's 8081 🤔